### PR TITLE
Convert Water Meter values

### PR DESCRIPTION
**Pull request by tizzen33**
Hi Frenck

This pull request converts water usage sensors to more readable values:

- Average: liters/hour to liters/minute
- Current: liters/hour to liters/minute
- Day Average: cm3 to m3
- Day Usage: cm3 to m3
- Meter: cm3 to m3

I also added Daily Water Cost which is also sent by the API.

### CODE REVIEW

File: toonapi/models.py [hunk 1]
```diff
@@ -366,16 +369,18 @@ class WaterUsage:
...
-        self.current = process_data(data, "value", self.current)
-        self.day_average = process_data(data, "avgDayValue", self.day_average)
-        self.day_usage = process_data(data, "dayUsage", self.day_usage)
+        self.average = process_data(data, "avgValue", self.average, convert_lmin)
+        self.current = process_data(data, "value", self.current, convert_lmin)
+        self.day_average = process_data(
+            data, "avgDayValue", self.day_average, convert_m3
+        )
+        self.day_cost = process_data(data, "dayCost", self.day_cost)
+        self.day_usage = process_data(data, "dayUsage", self.day_usage, convert_m3)
```
Comment (frenck) [inline toonapi/models.py:378]: Thought: For day usage, the M3 might be too big? Considering the average water usage for a household in NL to be somewhere 90-150M3 a year?

Comment (tizzen33) [reply toonapi/models.py:378]: True, but I tought I'd follow the same unit as the Boxx display.

Comment (frenck) [reply toonapi/models.py:378]: Ok that makes sense 👍

**Review by frenck — CHANGES_REQUESTED** [review]
Ok, in general 👍  Great job!

One small change is still needed. The types of the class variables of `WaterUsage` are not correct now anymore. `average`, `current`, `day_average`, `day_usage` and `meter` are now a `float` because of the calculations.

The typehints for those are still `int`.

**Review by frenck — APPROVED** [review]
Awesome! I will patch up a package release today 👍

**Comment by tizzen33** [comment]
Ok, great!

**Pull request merged**